static/_redirects

```diff
@@ -109,23 +109,22 @@
 /guides/artifacts/artifacts-core-concepts /guides/artifacts 301
 /guides/artifacts/artifacts-faqs /support/index_artifacts 301
 /guides/artifacts/dataset-versioning /guides/artifacts 301
-/guides/artifacts/model-versioning /guides/model_registry 301
+/guides/artifacts/model-versioning /guides/core/registry/model_registry 301
 /guides/artifacts/quickstart /guides/artifacts/artifacts-walkthrough 301
 /guides/artifacts/references /guides/artifacts 301
 /guides/artifacts/using-artifacts /guides/artifacts/download-and-use-an-artifact 301
 /guides/comparing-runs /guides/runs 301
 /guides/config /guides/sweeps/define-sweep-configuration 301
 /guides/configurations /guides/track/config 301
-/guides/data-and-model-versioning / 301
-/guides/core/tables /guides/models/tables 301
-/guides/data_types /ref/python/data-types 301
+/guides/core/tables/* /guides/models/tables/:splat 301
 /guides/data-and-model-versioning/dataset-versioning /guides/artifacts 301
 /guides/data-and-model-versioning/intro / 301
-/guides/data-and-model-versioning/model-versioning /guides/model_registry 301
+/guides/data-and-model-versioning/model-versioning /guides/core/registry/model_registry 301
```

Comment:
Not working, goes to https://docs-1245.docodile.pages.dev/guides/models/tables/registry/model_registry